src/plugins/auto_batch/src/plugin.cpp

```diff
@@ -19,7 +19,7 @@
 namespace ov {
 namespace autobatch_plugin {
 
-std::vector<std::string> supported_configKeys = {ov::device::priorities.name(), ov::auto_batch_timeout.name()};
+std::vector<std::string> supported_configKeys = {ov::device::priorities.name(), ov::auto_batch_timeout.name(), "PERF_COUNT", "NUM_STREAMS"};
```

Comment:
Change to API 2.0 property
NUM_STREAMS -> ov::num_streams.name()
PERF_COUNT -> ov::enable_profiling.name()